awx/main/tests/unit/models/test_survey_models.py

```diff
@@ -132,29 +132,6 @@ def test_survey_passwords_not_in_extra_vars():
     }
 
 
-def test_job_safe_args_redacted_passwords(job):
```

Comment:
Are we still verifying this behavior elsewhere?

Reply:
It's not a thing. We don't pass extra vars via -e, we embed them in files and then -e the file.